src/main/java/Ruby.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Main class for the chatbot named Ruby.
+ * Ruby assists users in recording and organizing their daily tasks.
+ * It interacts with users through the command line, processing commands to manage tasks.
+ */
+import java.util.Scanner;
+
+public class Ruby {
+    private static final Scanner in  = new Scanner(System.in);
+    private static TaskList t1 = new TaskList();
+    private static String userInput; //User command input through the command line
+
+
+    public static void main(String[] args) {
+        greet();
+        processor();
+        exit();
+    }
+
+    /**
+     * Prints a greeting message to the user.
+     */
+    private static void greet(){
+        print(new String[]{"Hi, I am here. Greets from Ruby.", "What can I do for you?"});
+    }
+
+    /**
+     * Prints an exit message and terminates the chatbot.
+     */
+    private static void exit(){
+        print("Bye. Always feels good work with you.");
+    }
+
+    /**
+     * Main processing loop for user commands.
+     * Continuously captures and processes user input until the "bye" command is issued.
+     */
+    private static void processor(){
+        String [] userInputs = inputCatcher();
```

Comment:
```suggestion
        String[] userInputs = inputCatcher();
```

Perhaps you could remove the space for code consistency